django/utils/archive.py

```diff
@@ -216,11 +216,10 @@ def close(self):
         self._archive.close()
 
 
-extension_map = {
-    '.tar': TarArchive,
-    '.tar.bz2': TarArchive,
-    '.tar.gz': TarArchive,
-    '.tgz': TarArchive,
-    '.tz2': TarArchive,
-    '.zip': ZipArchive,
-}
+extension_map = dict.fromkeys((
+    '.tar',
+    '.tar.bz2', '.tbz2', '.tbz', '.tz2',
```

Comment:
Perhaps this refactoring and the addition of additional extensions like tbz2 should be done in a separate commit to highlight that they don't have any tests or documentation.

Reply:
Ok.